src/app/Containers/RecommendedMoviesContainer/index.unit.test.js

```diff
@@ -0,0 +1,51 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+import MovieListContainer from '../MovieListContainer';
+import { MovieList } from '../../Components';
+import { SUCCESS } from '../../../common/constants';
+import Movie from '../../Components/Movie';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+const getContainerWithSpecificState = (state) => {
+  const store = mockStore(state);
+  const container = mount(
+    <Provider store={store}>
+      <Router>
```

Comment:
It's a good idea to create a helper to do this wrapping of Provider and Router, since it's repeated several times